yb_migrate/src/srcdb/mysql.go

```diff
@@ -0,0 +1,53 @@
+package srcdb
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/go-sql-driver/mysql"
+)
+
+type MySQL struct {
+	source *Source
+
+	db *sql.DB
+}
+
+func newMySQL(s *Source) *MySQL {
+	return &MySQL{source: s}
+}
+
+func (ms *MySQL) Connect() error {
+	db, err := sql.Open("mysql", ms.getConnectionString())
```

Comment:
i think we can not using this connection outside the Connect() function.
so its good to have `defer db.Close()`

Reply:
similarly for other sources

Reply:
That's intentional. We want all source db access to happen behind the `SourceDB` implementation. Our eventual plan is to encapsulate ALL source db access inside the `srcdb` package.

That will make it easy to add support for other databases.